examples/data/icp_data/force4DOFForPointToPlaneMinimizer.yaml

```diff
@@ -0,0 +1,53 @@
+readingDataPointsFilters:
+  - SurfaceNormalDataPointsFilter:
+      knn: 12
+      epsilon: 1.33
+      keepNormals: 1
+      keepDensities: 1
+
+referenceDataPointsFilters:
+  - SurfaceNormalDataPointsFilter:
+      knn: 15
+
+  - ObservationDirectionDataPointsFilter
+
+  - OrientNormalsDataPointsFilter:
+      towardCenter: 1
+
```

Comment:
Are the extra lines between the reference data points filters intended?

Reply:
I like it better, easier to distinguish between the filters.